src/io/flutter/vmService/VmServiceWrapper.java

```diff
@@ -421,19 +448,64 @@ void received(List<Breakpoint> breakpointResponses, List<RPCError> errorResponse
   }
 
   public void removeBreakpoint(@NotNull final String isolateId, @NotNull final String vmBreakpointId) {
-    addRequest(() -> myVmService.removeBreakpoint(isolateId, vmBreakpointId, VmServiceConsumers.EMPTY_SUCCESS_CONSUMER));
+    addRequest(() -> myVmService.removeBreakpoint(isolateId, vmBreakpointId, new RemoveBreakpointConsumer() {
+      @Override
+      public void onError(RPCError error) {
+
+      }
+
+      @Override
+      public void received(Sentinel response) {
+
+      }
+
+      @Override
+      public void received(Success response) {
+
+      }
+    }));
   }
 
   public void resumeIsolate(@NotNull final String isolateId, @Nullable final StepOption stepOption) {
     addRequest(() -> {
       myLatestStep = stepOption;
-      myVmService.resume(isolateId, stepOption, null, VmServiceConsumers.EMPTY_SUCCESS_CONSUMER);
+      myVmService.resume(isolateId, stepOption, null, new ResumeConsumer() {
```

Comment:
can we update EMPTY_SUCCESS_CONSUMER or provide a new empty one for this case and the one starting on line 493?